homeassistant/components/camera/onvif.py

```diff
@@ -183,3 +196,23 @@ def handle_async_mjpeg_stream(self, request):
     def name(self):
         """Return the name of this camera."""
         return self._name
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        attrs = super().device_state_attributes
```

Comment:
Remove this. The parent class doesn't return any attrs.

Reply:
For now. What if it will someday be modified? Further modifications would be needed.

Reply:
The idea is that the child class should populate this property. The entity base class only implements this as a template. It's pretty clear from the docstring too:
```
"""Return device specific state attributes.
Implemented by platform classes.
"""
```